src/Controller/EnMarche/SummaryManagerController.php

```diff
@@ -86,4 +89,49 @@ public function removeExperienceAction(Request $request, JobExperience $experien
 
         return $this->redirectToRoute('app_summary_manager_index');
     }
+
+    /**
+     * @Route("/skills", name="app_summary_manager_handle_skills")
+     * @Method("GET|POST")
+     */
+    public function handleSkillAction(Request $request)
```

Comment:
Should be named `handleSkillsAction` but it does not matters since we'll remove this action once my other PR will be merged with one action for all steps.